R/layers2traces.R

```diff
@@ -603,10 +626,21 @@ geom2trace.GeomBoxplot <- function(data, params, p) {
 
 #' @export
 geom2trace.GeomText <- function(data, params, p) {
+  text <- as.character(data[["label"]])
+  text <- ifelse(
+    grepl("bold", data[["fontface"]]),
+    paste0("<b>", text, "</b>"),
+    text
+  )
+  text <- ifelse(
+    grepl("italic", data[["fontface"]]),
+    paste0("<i>", text, "</i>"),
+    text
+  )
```

Comment:
I was using `ifelse` as I was anticipating `data[["fontface"]]` could be a vector (I think it can be specified via `aes`).  Is this an incorrect assumption?